.github/workflows/publish.yml

```diff
@@ -8,19 +8,16 @@ jobs:
   publish-npm:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout the project
-        uses: actions/checkout@v4
-      - name: Use Node.js 16.x (LTS)
-        uses: actions/setup-node@v4
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: 16.x
-          registry-url: https://registry.npmjs.org/
-          cache: npm
```

Comment:
For small projects, the cache will probably just cause thrashing, which is just a waste of compute resources.